common/models/application.js

```diff
@@ -72,12 +72,13 @@ function generateKey(hmacKey, algorithm, encoding) {
 
 module.exports = function(Application) {
   // Workaround for https://github.com/strongloop/loopback/issues/292
+  Application.definition.rawProperties.created.default =
   Application.definition.properties.created.default = function() {
     return new Date();
   };
```

Comment:
These five lines (starting with the "workaround" comment) should be removed, since the default property values is already configured via JSON, right?